nilearn/datasets.py

```diff
@@ -26,6 +25,17 @@
 
 import nibabel
 
+# Define _get_time, which is used module-wide to get the current time.
+try:
+    # Use uptime (when available) to avoid reporting time while computer slept.
+    import uptime
+    if uptime.uptime() is None:
+        raise Exception("uptime is not supported by this OS.")
+    _get_time = uptime.uptime
```

Comment:
I am somewhat agnostic, but I would suggest that the data fetching part of this package could really be an attractive aspect of it.  I think having excellent download tools is going to be extremely important for the USA's BRAIN initiative (among others; it's the one I'm most familiar with) and the current push for data sharing.

I plan to post an issue to discuss later, but I think that the `nilearn` dataset functionality can be used as a starting point for something much bigger.  This is something I would be happy to work on, and I think I have some contacts that could help get some traction on it.  For this reason, I suggest keeping this kind of code in--I think it'll become more important over time.

With that said, it's such a simple snippet, I would only mention this once now and leave it to others to decide what's the right way to go!